orbit/cmd/orbit/orbit.go

```diff
@@ -267,6 +273,15 @@ func main() {
 		g.Add(systemChecker.Execute, systemChecker.Interrupt)
 		go osservice.SetupServiceManagement(constant.SystemServiceName, systemChecker.svcInterruptCh, appDoneCh)
 
+		// periodically run launchctl kickstart -k softwareupdated on macOS
+		if runtime.GOOS == "darwin" && !c.Bool("disable-kickstart-softwareupdated") {
+			const softwareUpdatedKickstartInterval = 12 * time.Hour
```

Comment:
Thoughts on the interval? Note that it will run ~immediately first, and then at this interval.